src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
+@import './mixins';
+
+$content-width: 166px;
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+.cards {
+  border: 1px solid #f3f3f3;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  margin: 0 auto;
+  display: flex;
+  box-sizing: border-box;
```

Comment:
it's allowed to move only this rule to `*` seelctor

Reply:
is it sur ?

Everything goes wrong beacause every elements has width : 200 px, can i keep this proporties in my .cards style?